docs/tutorials/05-external-jsonschema.md

```diff
@@ -361,7 +363,7 @@ spec:
       repositoryContext:
         type: ociRegistry
         baseUrl: eu.gcr.io/gardener-project/landscaper/tutorials/components
-      componentName: github.com/gardener/landscaper/external-jsonscheme/echo-server
+      componentName: github.com/gardener/landscaper/external-jsonschema/echo-server
```

Comment:
The component descriptor at `./docs/tutorials/resources/external-jsonschema/echo-server/component-descriptor.yaml` still has _external-jsonschem**e**_ in it. Can you update that as well?

Reply:
Good catch. Done.